run_tests.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+import subprocess
+import argparse
+import time
+
+def run_python_tests(coverage=False):
+    """Run python unit tests."""
+    try:
+        if coverage:
+            subprocess.check_call(
+                'nosetests --with-coverage'
+                + ' --cover-package=timesketch_api_client,timesketch'
+                + ' api_client/python/timesketch_api_client/ timesketch/',
+                shell=True,
+            )
+        else:
+            subprocess.check_call(['nosetests'])
+    finally:
+        subprocess.check_call(['rm', '-f', '.coverage'])
+
+def run_python_linter():
+    """Run pylint on timesketch and api_client."""
+    subprocess.check_call(['pylint', 'timesketch'])
```

Comment:
Will this use our utils/pylintrc?

Reply:
Yes, because utils/pylintrc was moved to <project_root>/.pylintrc

Reply:
Great!